runtime/src/main/java/org/corfudb/runtime/exceptions/StaleTokenException.java

```diff
@@ -0,0 +1,16 @@
+package org.corfudb.runtime.exceptions;
+
+import lombok.Getter;
+
+/**
+ * Created by dmalkhi on 7/21/17.
+ */
+public class StaleTokenException extends RuntimeException {
+    @Getter
```

Comment:
Should staleEpochException extend wrongepochexception (is stale epoch a more specific type of wrongepoch?)